assets/css/live_view.css

```diff
@@ -65,3 +65,14 @@
   background: #ffe6f0!important;
 }
 
+/* Loading states */
+.phx-click-loading {
```

Comment:
We can remove these classes, right? They were just for trying things out?

Reply:
Also, should we delete this whole CSS file and just push people to use NProgress?

Reply:
Yes, this file at the moment is imported in the demo app and used for testing things out in a versioned way. We can kill it once the --live generators are released since we'll have a versioned place for these things to live